hadoop-ozone/recon/pom.xml

```diff
@@ -86,7 +86,7 @@
       <plugin>
         <groupId>com.github.eirslett</groupId>
         <artifactId>frontend-maven-plugin</artifactId>
-        <version>1.10.0</version>
+        <version>${frontend-maven-plugin.version}</version>
```

Comment:
Nit: version is inherited from parent pom's `pluginManagement` section, no need to explicitly specify here.  But it does no harm and can be removed later, it's more important to merge this speed improvement sooner than later.